Results: figure 2 (gene set comparison using expression data only) (#10)
greenelab/mpmp-manuscript: content/04.results.md

**C.** Schematic of evaluation pipeline.
](images/figure_1.png){#fig:overview}

### Selection of cancer-related genes improves predictive signal

As a baseline, we evaluated prediction of mutation status from gene expression data across several different gene sets.
Past work has evaluated mutation prediction for the top 50 most mutated genes in TCGA [@doi:10.1186/s13059-020-02021-3], and we sought to extend this to a broader list of gene sets.
We compared a set of cancer-related genes from Vogelstein et al. 2013 [@doi:10.1126/science.1235122] with a set of random genes having equal size, and a set of the most mutated genes in TCGA having equal size.
For all gene sets, we used only the set of TCGA samples for which both gene expression and somatic mutation data exists, resulting in a total of 9,074 samples from all 33 cancer types (Figure {@fig:expression_gene_sets}A).
This set of samples was further filtered for each target gene to cancer types containing at least 15 mutated samples and at least 5% of samples mutated for that cancer type.

> We then followed the evaluation procedure described above for each target gene in each of the three gene sets.

Reviewer: ```suggestion
We then evaluated the performance for each target gene in each of the three gene sets.
```

Figure {@fig:expression_gene_sets}B summarizes performance for all genes in each gene set.

> Across all genes, the Vogelstein et al. gene set markedly improves the aggregate performance compared to the randomly chosen genes, and compared to choosing genes based on their total mutation count.

Reviewer: ```suggestion
Genes from the Vogelstein et al. set were more predictable than randomly chosen genes or those selected by total mutation count.
```

Reviewer: I think avoiding the word "performance" more than is necessary will be helpful. A key is that this manuscript is not presenting a new method but instead comparing properties of data types and gene sets.
